appserver/jdbc/jdbc-runtime/src/main/java/org/glassfish/jdbc/util/JdbcResourcesUtil.java

```diff
@@ -67,27 +67,20 @@
  */
 public class JdbcResourcesUtil {
 
-    private volatile static JdbcResourcesUtil jdbcResourcesUtil;
-    private static Logger _logger = LoggerFactory.getLogger(JdbcResourcesUtil.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(JdbcResourcesUtil.class);
+
+    private static final JdbcResourcesUtil instance = new JdbcResourcesUtil();
     private ConnectorRuntime runtime;
 
     private JdbcResourcesUtil() {
     }
 
     public static JdbcResourcesUtil createInstance() {
-        //stateless, no synchronization needed
-        if(jdbcResourcesUtil == null){
-            synchronized(JdbcResourcesUtil.class) {
-                if(jdbcResourcesUtil == null) {
-                    jdbcResourcesUtil = new JdbcResourcesUtil();
-                }
-            }
-        }
-        return jdbcResourcesUtil;
+        return instance;
```

Comment:
The instance has basically no costs - there is no need to make it lazy or complicated.